exporter/loggingexporter/logging_exporter.go

```diff
@@ -25,18 +28,128 @@ import (
 	"github.com/open-telemetry/opentelemetry-collector/exporter/exporterhelper"
 )
 
+type traceDataBuffer struct {
+	str strings.Builder
+}
+
+func (b *traceDataBuffer) logEntry(format string, a ...interface{}) {
+	b.str.WriteString(fmt.Sprintf(format, a...))
+	b.str.WriteString("\n")
+}
+
+func (b *traceDataBuffer) logAttr(label string, value string) {
+	b.logEntry("    %-12s: %s", label, value)
+}
+
+func (b *traceDataBuffer) logMap(label string, data *map[string]string) {
+	if data == nil {
```

Comment:
I look at the callsites and noone is calling logMap with nil second parameter, so I am confused as to what the intent of the code is. Please clarify.

Reply:
It should have just checked if the map is empty. Fixed that